### PR TITLE
#160609505 Set-up react app

### PR DESCRIPTION
**Pull request by mendozabree**
#### What does this PR do?

Create a react app
#### Description of Task to be completed?

Currently, there is no react app setup for our codebase and no linting.

This task creates the react app for our codebase and adds linting using eslint.

#### How should this be manually tested?

- Run `npm install` to install all dependencies
- Run `npm start`

#### What are the relevant pivotal tracker stories?

[#160609505](https://www.pivotaltracker.com/story/show/160609505)

#### Any background context you want to add?
N/A

#### Screenshots
screenshot 2018-10-29 at 16 54 40

### CODE REVIEW

**Review by samuelbwambale — APPROVED** [review]
LGTM

**Review by RonKbS — APPROVED** [review]
LGTM

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (kakaemma) [inline public/index.html]: Kindly look into changing the title to reflect our project.

**Review by dondrzzy — COMMENTED** [review]
Is the `react logo.svg` still required?

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

File: public/index.html (hunk 1 above)
Comment (mendozabree) [reply public/index.html]: Thank you @kakaemma, let me change that.

**Comment by mendozabree** [comment]
@dondrzzy, I have removed the `logo.svg` file

**Pull request merged**